### DISCUSSION TITLE
Bindings for a scripting language ?

### DISCUSSION THREAD

**Discussion by llothar** [ArthurSonzogni/FTXUI#799] [Ideas]
I would love to have Ruby and Python bindings.
I think this could give another productivity boost.

**Comment by ArthurSonzogni** [comment]
This sounds a bit difficult, because FTXUI is made to have a "nice" syntax adapted to C++. We would have something a inferior when translated in a different language.

Python has `Rich` library, which appears to be really nice. Maybe it is not worth adding FTXUI. WDYT?
